src/core/client/base.js

```diff
@@ -30,6 +30,14 @@ export default async function createClient(
     await fetchBufferedLogs();
   }
 
+  const strictModeEnabled = convertBoolean(config.get('enableStrictMode'));
```

Comment:
I don't think we need a call to `convertBoolean`, you can probably remove it and inline `config.get('enableStrictMode')` in the `if` statement.

Reply:
I have refactored and made the changes.